nvtabular/ops/target_encoding.py

```diff
@@ -215,6 +212,9 @@ def dependencies(self):
         return self.dependency
 
     def output_column_names(self, columns):
+        if hasattr(self.target, "output_columns"):
+            self.target = self.target.output_columns
+
```

Comment:
This should move to an override of a new `fit_schema` method on the `Operator` base class.